builder/builder-next/controller.go

```diff
@@ -189,6 +190,10 @@ func newController(rt http.RoundTripper, opt Opt) (*control.Controller, error) {
 		ResolveCacheExporterFuncs: map[string]remotecache.ResolveCacheExporterFunc{
 			"inline": inlineremotecache.ResolveCacheExporterFunc(),
 		},
+		Entitlements: []string{
+			string(entitlements.EntitlementNetworkHost),
+			// string(entitlements.EntitlementSecurityInsecure),
```

Comment:
Should I enable `EntitlementSecurityInsecure` right now? I'm inclined to suggest yes.

Reply:
Should we enable it thru config or at least envvar maybe? I guess the usecase is: someone who has access to /build endpoint ONLY wants to run builds sandboxed.

Reply:
Even if we enable it there is no way to allow it through `/build` atm. But it would be possible to allow it through the new grpc endpoint. If we force the user to restart, it limits the usage. It's not like this would be the first privileged endpoint in the api. If in the config, then I think the default should still be to enable this.

Reply:
I think we can disable it for now

Reply:
I guess you're right that the API itself is insecure already so this is more about enabling a feature than compromising any false sense of security. Users who need an auth proxy in front, won't be affected. In that case we could enable it. @AkihiroSuda WDYT?

Reply:
I can open up a separate PR for this where we can discuss more if needed.